### ISSUE TITLE
Is there an error in transformations?

### ISSUE THREAD

**Issue by huanglianghua** [bilylee/SiamFC-TensorFlow#56]
Hi @bilylee , thanks for your fantastic work.

In the following code ([here](https://github.com/bilylee/SiamFC-TensorFlow/blob/17f51563f081281fc7d966602d393715e7ecb469/datasets/dataloader.py#L30)):

```
self.z_transform = Compose([RandomStretch(),
                                  CenterCrop((255 - 8, 255 - 8)),
                                  RandomCrop(255 - 2 * 8),
                                  CenterCrop((127, 127))])
self.x_transform = Compose([RandomStretch(),
                                  CenterCrop((255 - 8, 255 - 8)),
                                  RandomCrop(255 - 2 * 8), ])
```

The output `z` is `127 x 127`, but output `x` is `239 x 239` not `255 x 255`.

The response size with `239 x 239` is `15 x 15` but with `255 x 255` is `17 x 17`.

`17 x 17` is correct since it is exactly same as the response size in the inference stage.

Is this an error or so? (Although this will not report any error during training since the label size is directly deduced from response size!).

**Comment by shuida** [comment]
I have the same problem. Has it been solved?

**Comment by bilylee** [comment]
Hi,

During training, image patch with size `239 x 239` is cropped from the `255 x  255` input image for translation data augmentation. It won't affect tracking, since the Siamese network learns a similarity metric that is position agnostic.

Please also refer to #32

**Comment by shuida** [comment]
ok，thank you.
